h2o-webserver-iface/src/main/java/water/webserver/iface/WebServer.java

```diff
@@ -0,0 +1,12 @@
+package water.webserver.iface;
+
+/**
+ * All the functionality that we need to call on an existing instance of HTTP server (servlet container).
```

Comment:
I suggest removing the `(servlet container)` suffix. There could be anything inside.

Reply:
I don't understand the statement "There could be anything inside" because, there cannot. This is API documentation and as such, it is normative, not descriptive - it explains what we expect (and require) here. If the implementer puts here anything else, it is not going to work in expected ways. 
Or, maybe I misunderstood the note?

Reply:
in fact, we really *do* require something that understands servlets; as a user of this API, you pass some servlets to it in the `H2OHttpView`